### PR TITLE
Added joint position telemetry to robot

### PR DESCRIPTION
**Pull request by DrDab**

### CODE REVIEW

File: src/network/MissionControlProtocol.cpp [hunk 1]
```diff
@@ -351,6 +371,25 @@ void MissionControlProtocol::jointPowerRepeatTask() {
 	}
 }
 
+void MissionControlProtocol::jointPosReportTask() {
+	dataclock::time_point pt = dataclock::now();
+
+	while (true) {
+		for (auto cur = robot::types::name_to_jointid.begin();
+			 cur != robot::types::name_to_jointid.end(); cur++) {
```
Comment (abhaybd) [inline src/network/MissionControlProtocol.cpp:379]: Can you replace this with:
```c++
for (const auto& cur : robot::types::name_to_jointid) {
    ...
}
```

Which is shorter and cleaner

File: src/network/MissionControlProtocol.cpp [hunk 2]
```diff
@@ -351,6 +371,25 @@ void MissionControlProtocol::jointPowerRepeatTask() {
...
+			robot::types::DataPoint<int32_t> jpos = robot::getJointPos(cur->second);
+			if (jpos.isValid()) {
+				auto frozenStr = cur->first;
+				std::string str(frozenStr.data(), frozenStr.size());
```
Comment (abhaybd) [inline src/network/MissionControlProtocol.cpp:383]: Can these names be more descriptive? Maybe `jointNameFrozenStr` and `jointNameStr`?

**Review by abhaybd — CHANGES_REQUESTED** [review]
Mostly pretty good! Just some minor changes needed.

**Comment by DrDab** [comment]
Done!

**Review by abhaybd — APPROVED** [review]
Looks good!

**Pull request merged**